src/scipp/coords/coord_table.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+# @author Jan-Lukas Wynen
+
+import dataclasses
+from typing import Dict, Iterable, List, Set, Tuple
+
+from .coord import Coord, Destination
+from .options import Options
+from .rule import FetchRule, RenameRule, Rule, rule_output_names
+
+
+class CoordTable:
```

Comment:
Add docstring explaining what this is used for?

Reply:
Thanks for the reminder! I forgot that I didn't write them yet.